ItemCatalogService.gRPCService/Startup.cs

```diff
@@ -14,14 +15,20 @@ namespace ItemCatalogService.gRPCService
 {
     public class Startup
     {
+        public IConfiguration Configuration { get; }
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddGrpc();
-            
-            var connectionString = Environment.GetEnvironmentVariable("DB_CONNECTION_STRING");
-            services.AddDbContext<ItemContext>(options => options.UseNpgsql(connectionString));
+
+            services.AddDbContext<ItemContext>(options => options.UseNpgsql(Configuration["ConnectionStrings:DevConnectionPlSql"]));
```

Comment:
```suggestion
            services.AddDbContext<ItemContext>(options => options.UseNpgsql(Configuration.GetConnectionString("DevConnectionPlSql")));
```

Reply:
For this the Connections string in appsettings.Development has to look like 
`"ConnectionStrings": {
      "DevConnectionPlSql": connectionString
}`

Reply:
Look here [https://stackoverflow.com/questions/44931613/how-to-correctly-store-connection-strings-in-environment-variables-for-retrieval](url)